embedfs/embed_test.go

```diff
@@ -184,30 +178,23 @@ func TestReadDir(t *testing.T) {
 		wantErr bool
 	}{
 		{
-			name: "singleFile",
+			name: "testdata",
 			path: "testdata",
-			fs:   &singleFile,
-			want: []string{"empty.txt"},
+			fs:   testdata.GetTestData(),
+			want: []string{"empty.txt", "file1.txt", "file2.txt", "subdir"},
 		},
 		{
-			name:    "empty",
+			name:    "empty path",
 			path:    "",
-			fs:      &empty,
-			want:    []string{},
-			wantErr: true,
+			fs:      testdata.GetTestData(),
+			want:    []string{"testdata"},
+			wantErr: false,
```

Comment:
This is changing the meaning of the test. Let's revert this and change `wantErr` to `false`.

```suggestion
			name:    "empty FS",
			path:    "",
			fs:      &empty,
			want:    []string{},
			wantErr: false,
```